lib/packwerk/configuration.rb

```diff
@@ -3,6 +3,7 @@
 
 require "pathname"
 require "yaml"
+require "packwerk/formatters/offenses_formatter"
```

Comment:
Done! I had to put it at the end of the file otherwise autoloads wouldn't yet be registered, causing uninitialized constant errors. Let me know what ya think.